types/proto.go

```diff
@@ -1,10 +1,5 @@
 package types
 
-import (
-	_ "github.com/gogo/protobuf/gogoproto"    // nolint
```

Comment:
@aaronc  is this here for a reason?

Reply:
@marbar3778 I would block on this until we hear from @aaronc @clevinson or @alexanderbez, it might be there for a reason.

Reply:
agree

Reply:
I believe I added this as a workaround to having these bundled in `go.mod`. I believe `github.com/gogo/protobuf/gogoproto` can be removed since we use it directly elsewhere.

However, I still think we need to keep `github.com/regen-network/cosmos-proto`. Please correct me if I'm wrong @aaronc

Reply:
We need them to stay in go.mod. l think both should be referenced elsewhere by now. We can try and see if go.mod changes

Reply:
the go.mod isnt changing and the linter would crash if there was an unused module in the go.mod so it looks to be good